src/Controllers/MoviesController.cs

```diff
@@ -70,7 +71,7 @@ public async Task<IActionResult> GetMoviesAsync([FromQuery] MovieQueryParameters
                 res = await ResultHandler.Handle(dal.GetMoviesAsync(movieQueryParameters), Logger).ConfigureAwait(false);
 
                 // use cache dal on Cosmos 429 errors
-                // todo - this will never get called
+                // todo - this will never get called - code review
                 //        if App.Config.Cache, the cache will be used in ResultHandler
```

Comment:
This was an idea of a scenario we had (only cache if the Cosmos query fails)

We're not using it and I'm not sure that we ever will. It doesn't currently work. To fix, it would require another command line flag.

Should we add an item to fix it or remove from the code?